web/src/beta/ui/fields/InputSelectField.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Selector,
+  SelectorProps,
+  TextInput,
+  TextInputProps
+} from "@reearth/beta/lib/reearth-ui";
+import { styled } from "@reearth/services/theme";
+import { FC, useState } from "react";
+
+import CommonField, { CommonFieldProps } from "./CommonField";
+
+export type InputFieldProps = CommonFieldProps &
+  Pick<
+    TextInputProps,
+    "value" | "placeholder" | "onChange" | "onBlur" | "disabled" | "appearance"
+  > &
+  Pick<SelectorProps, "displayValue"> & { sketchCustomProperties?: string[] };
```

Comment:
```suggestion
  Pick<SelectorProps, "displayValue"> & { options?: {label?:string; value:string}[] };
```
Keep the option type the same with `Selector`